.github/workflows/torch-workflow-archiver-nightly-build.yml

```diff
@@ -1,26 +0,0 @@
-name: Push torch-workflow-archiver-nightly
```

Comment:
Why this workflow is deleted?

Reply:
Merged this workflow to the torchserve-nightly-build to save resources